app/controllers/application_controller.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
-
   before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
before_action duplication on line 4 and 13

Reply:
fixed too